src/call.cpp

```diff
@@ -957,6 +955,19 @@ char * call::get_last_header(const char * name)
         ERROR("call::get_last_header: Header to parse bigger than %d (%zu)", MAX_HEADER_LEN, strlen(name));
     }
 
+    if(strcmp(name, "media_ip")==0) {
```

Comment:
I wonder if it might be slightly better to always call `extract_rtp_remote_addr` (at line 2820 or so) instead of guarding it with `#ifdef RTP_STREAM`, and store the host/port off as member variables on the `call` object, then just use those (in the same way we use the pre-stored `peer_tag` when handling `E_Message_Peer_Tag_Param`). That would guarantee that the host/port would be retrieved consistently whether you used built-in RTP streaming or this method.